clusterloader2/pkg/prometheus/manifests/prometheus-prometheus.yaml

```diff
@@ -17,8 +19,13 @@ spec:
   replicas: 1
   resources:
     requests:
+      {{if $PROMETHEUS_SCRAPE_KUBELETS}}
+      # TODO(oxddr): figure out memory limit
+      memory: 10Gi # {{MultiplyInt 2 (AddInt 1 (DivideInt .Nodes 2000))}}Gi
```

Comment:
10Gi means that it requires a heapster node that is at least n1-standard-4. We don't set-up heapster node for gce-100 tests, please document that.